### DISCUSSION TITLE
Issue with Creating soca-jobs* Index for SOCA Dashboard

### DISCUSSION THREAD

**Discussion by wbadyx** [awslabs/scale-out-computing-on-aws#128] [Q&A] [answered]
Hello,

On the SOCA web page under "My Activity," I am getting the following message:
"Unable to retrieve index ID for soca-jobs*. To do the initial setup, follow instructions available on https://awslabs.github.io/scale-out-computing-on-aws/analytics/monitor-cluster-activity/"

However, when I follow the instructions in the documentation, I am unable to create the soca-jobs* index. Instead, I can only create the soca_jobs* index, which results in the Dashboard not displaying correctly.

Could you provide more information on what might be causing this issue? I am not sure where the problem lies.

Thank you for your help.

**Comment by mcrozes** [comment, marked as answer]
Hello,

 I assume you are using OpenSearch and not ElasticSearch. It seems we haven't had the chance to update `my_files` for OpenSearch yet per. https://github.com/awslabs/scale-out-computing-on-aws/blob/main/source/soca/cluster_web_ui/views/my_activity.py#L62-L79 .
 
 Since it's a defect, can you please create a Github Issue and I will look into our options here. (aka: query OpenSearch JobIndex API to list them). 

For now, as you mentioned, `my_activity` will not work for you but you can still get all data directly on your OpenSearch dashboard.

**Reply by wbadyx** [reply]
Thank you for your reply. Have a good day!

**Comment by mcrozes** [comment]
Closing as this will be tracked on https://github.com/awslabs/scale-out-computing-on-aws/issues/129
